movie-service/src/main/java/com/barbenheimer/movieservice/serviceImpl/CustomerDetailServiceImpl.java

```diff
@@ -19,14 +19,14 @@ public CustomerDetailServiceImpl(CustomerDetailRepository customerDetailReposito
         this.customerDetailRepository = customerDetailRepository;
     }
 
-    public void inputCustomerDetails(String email){
+    public CustomerDetail inputCustomerDetails(String email){
         CustomerDetail customerDetail = new CustomerDetail();
-        String regex = "^[A-Z0-9+_.-]+@[A-Z0-9.-]+$";
+        String regex = "^[a-zA-Z0-9+_.-]+@[a-zA-Z0-9.-]+$";
```

Comment:
Hi, we can actually use lombok to specify the validation constraints on the entity itself! Then we won't have to keep validating in our services 😃